test/iron-menu-behavior.html

```diff
@@ -61,7 +61,7 @@
           var menu = fixture('basic');
           menu.focus();
           setTimeout(function() {
-            assert.equal(document.activeElement, menu.firstElementChild, 'document.activeElement is first item')
+            assert.equal(document.activeElement, menu.firstElementChild, 'document.activeElement is first item');
```

Comment:
This was already broken before you got to it, but I think `document` needs to be wrapped in `Polymer.dom` for this assertion to be resilient.